modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/converter/KnowledgeBaseArticleDTOConverter.java

```diff
@@ -99,13 +99,15 @@ public KnowledgeBaseArticle toDTO(DTOConverterContext dtoConverterContext)
 					_assetTagLocalService.getTags(
 						KBArticle.class.getName(), kbArticle.getClassPK()),
 					AssetTag.NAME_ACCESSOR);
-				numberOfAttachments = Optional.ofNullable(
-					kbArticle.getAttachmentsFileEntries()
-				).map(
-					List::size
-				).orElse(
-					0
-				);
+
+				numberOfAttachments = 0;
+				List<FileEntry> fileEntries =
```

Comment:
@JanisZhang, the code looks good to me. could you please keep code format consistent with https://github.com/liferay-core-infra/liferay-portal/pull/2860/commits/d3fcb1036929f716e7bebd58ddc694c76e3ca763. After you push the changes, you may only execute ci:test:sf

Reply:
Hi @yuhai, force-pushed. Thanks!